app/javascript/react/components/Map/Map.tsx

```diff
@@ -59,24 +93,14 @@ const Map = () => {
     unit_symbol: unit_symbol,
   });
 
+  // Effects
   useEffect(() => {
-    dispatch(fetchFixedSessions({ filters }));
-  }, [dispatch, filters]);
-
-  const sessionsData = useSelector(selectSessionsData);
-
-  const mappedSessionsData: Session[] = sessionsData.map((session) => {
-    return {
-      id: session.id,
-      lastMeasurementValue: session.lastMeasurementValue,
-      point: {
-        lat: session.latitude,
-        lng: session.longitude,
-        streamId: session.streamId.toString(),
-      },
-    };
-  });
+    fixedSessionTypeSelected
+      ? dispatch(fetchFixedSessions({ filters }))
+      : dispatch(fetchMobileSessions({ filters }));
+  }, [dispatch, filters, selectedSessionType]);
 
+  // Callbacks
```

Comment:
Does it add value to us? In one line below we can see the `useCallback` call

Reply:
Same as https://github.com/HabitatMap/AirCasting/pull/660#discussion_r1633213995